robyn/__init__.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, file_object: str, config: Config = Config()) -> None:
         self.file_path = file_object
         self.directory_path = directory_path
         self.config = config
-
+        self.dependencies = {"ALL":{'request':Request, 'response':Response}}
```

Comment:
Hey @DarrenSeng 👋 

Great progress so far 😄 

One more suggestion - can you extract this dictionary to a custom data structure?
```


i.e. like self.dependencies = DependencyMap()

class DependencyMap:
     """ This will have a dictionary inside"""
     
 ```
 
 This is called composition.
 
 A good read will be something like this - https://python-patterns.guide/gang-of-four/composition-over-inheritance/

Reply:
So something like?
`class DependencyMap:
    def __init__(self):
        self.dep_dict = {"ALL":{'request':Request, 'response':Response}}

class Robyn:
    def __init__(self, file_object: str, config: Config = Config()) -> None:
        self.dependencies = DependencyMap()
`

Reply:
@DarrenSeng , yes. And then the dependencymap will have certain accessor methods.